src/features/click-to-load.js

```diff
@@ -628,6 +629,48 @@ function showExtraUnblockIfShortPlaceholder (shadowRoot, placeholder) {
     }
 }
 
+/**
+ * Hide the info text (and move the "Learn More" link) if the placeholder is too
+ * narrow.
+ * @param {ShadowRoot} shadowRoot
+ * @param {HTMLElement} placeholder Placeholder for tracking element
+ * @param {number} narrowWidth
+ *    Maximum placeholder width (in pixels) for the placeholder to be considered
+ *    narrow.
+ */
+function hideInfoTextIfNarrowPlaceholder (shadowRoot, placeholder, narrowWidth) {
+    const { width: placeholderWidth } = placeholder.getBoundingClientRect()
+    if (placeholderWidth > 0 && placeholderWidth <= narrowWidth) {
+        const buttonContainer =
+              shadowRoot.querySelector('.DuckDuckGoButton.primary')?.parentElement
+        const contentTitle = shadowRoot.getElementById('contentTitle')
+        const infoText = shadowRoot.getElementById('infoText')
+        /** @type {HTMLElement?} */
+        const innerDiv = shadowRoot.querySelector('.DuckDuckGoSocialContainer')
+        /** @type {HTMLElement?} */
+        const learnMoreLink = shadowRoot.querySelector('.learnMoreLink')
+
+        // These elements will exist, but this check keeps TypeScript happy.
+        if (!buttonContainer || !contentTitle || !infoText || !innerDiv ||
+            !learnMoreLink) {
+            return
+        }
+
+        // Remove the information text.
+        infoText.remove()
+        learnMoreLink.remove()
+
+        // Append the "Learn More" link to the title.
+        contentTitle.innerText += '. '
+        learnMoreLink.style.removeProperty('font-size')
+        contentTitle.appendChild(learnMoreLink)
+
+        // Improve margin/padding, to ensure as much is displayed as possible.
+        buttonContainer.style.removeProperty('margin')
+        innerDiv.style.height = 'calc(100% - 80px)'
```

Comment:
Could you tell a bit more about why the `calc(100% - 80px)` height?

Reply:
The existing height is `calc(100% - 30px);` and I needed it to be a bit shorter. So with trial and error I found `calc(100% - 80px);` looked about right. I'm not sure why the original height was `calc(100% - 30px);` though, that was before my time.

Reply:
I think I was the one that added `calc(100% - 30px);`, and it was to in order to fit the "Share feedback" link under the placeholder. That's why I found it odd to have `80px` here now. You got to 80px because of the site where Costas was testing? If so, I think we can continue with 30px, as that website has some particular cropping of the video area and we shouldn't be tailoring the styles based only on that.

Reply:
Fair enough, I have removed that for now :+1: